src/events/event.controller.ts

```diff
@@ -60,9 +61,9 @@ export class EventController {
    * @returns A promise that resolves to void.
   */
   @Post('order-created')
-  async subscribeToOrderEvent(@Body('data') order: OrderDTO): Promise<void> {
+  async subscribeToOrderEvent(@Body() order: OrderDTO): Promise<void> {
```

Comment:
you sure data is not required here? The last time I tested it did not work without it

Reply:
Since I never got to test with "real" events I assume this is correct